### PR TITLE
Create login front end page

### PR DESCRIPTION
**Pull request by Naphrum**
https://github.com/HTMElephant/open-kitchen/issues/3

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      </Grid>
+      <Grid item xs={6}>
+        <TextField
+          id="outlined-basic"
+          label="Password"
+          variant="outlined"
+          type="password"
+          onChange={(e) => setPassword(e.target.value)}
+          error={loginError}
+          helperText={handleLoginError()}
```
Comment (jacobdman) [inline src/components/Login/Login.js:50]: Does this need to invoke a function?

```suggestion
          helperText={loginError && "Invalid credentials, Please try again."}
```

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { useState, useContext } from "react";
+import AppContext from "../../context/AppContext";
+import { Typography, Button, TextField, Grid, Link } from "@mui/material";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [isError, setIsError] = useState(false);
```
Comment (jacobdman) [inline src/components/Login/Login.js:8]: This is not used - let's kill it

```suggestion
```

File: src/context/AppContext.js [hunk 3]
```diff
@@ -1,12 +1,52 @@
...
+  }, []);
+
+  const login = async (email, password) => {
+    try{
+
+      const response = await axios.post("/v1/login", {
+        email,
+        password,
+      });
+      console.log(response);
```
Comment (jacobdman) [inline src/context/AppContext.js:28]: yikos

File: src/context/AppContext.js [hunk 4]
```diff
@@ -1,12 +1,52 @@
...
+        localStorage.setItem("user", JSON.stringify(response.data));
+        setLoginError(false)
+        navigate("/")
+      } else {
+        console.error("Login failed.");
+        setLoginError(true)
+      }
+    } catch (err) {
+      setLoginError(true)
+      console.log(err)
```
Comment (jacobdman) [inline src/context/AppContext.js:40]:
```suggestion
      console.error(err)
```

**Review by jacobdman — COMMENTED** [review]
i have left some comments. Great work though I love it and you the same (equally)

**Review by jacobdman — APPROVED** [review]
Nice work!

**Review by KyleD8n — APPROVED** [review]
Nice job Nate Nate Nate

**Pull request merged**